R/module-helpers.R

```diff
@@ -8,7 +8,7 @@
 #' @param water_sector A character vector containing water sector information
 #' @param water_type A character vector containing water type information
 #' @param water_mapping A tibble with a mapping between \code{water.sector} and  \code{supplysector}
-#' @param GLU An optional character vector containing GLU information (only used for irrigation with mapped water types)
+#' @param GLU An optional character vector containing GLU information(only used for irrigation with mapped water types)
```

Comment:
Hahah...oops. I copied over some code from the old data system and did a find-and-replace to fix the GCAM spacing convention. Will fix.